packages/atomic/src/components/atomic-facet-manager/atomic-facet-manager.tsx

```diff
@@ -26,30 +29,38 @@ interface FacetElement extends HTMLElement {
 export class AtomicFacetManager implements InitializableComponent {
   @InitializeBindings() public bindings!: Bindings;
   private facetManager!: FacetManager;
+  public searchStatus!: SearchStatus;
+
+  @BindStateToController('searchStatus')
+  @State()
+  private searchStatusState!: SearchStatusState;
 
   @Element() private host!: HTMLDivElement;
 
   @BindStateToController('facetManager', {
-    onUpdateCallbackMethod: 'onFacetManagerUpdate',
+    onUpdateCallbackMethod: 'sortFacets',
   })
   @State()
   public facetManagerState!: FacetManagerState;
   @State() public error!: Error;
 
   public initialize() {
+    this.searchStatus = buildSearchStatus(this.bindings.engine);
     this.facetManager = buildFacetManager(this.bindings.engine);
-  }
 
-  public onFacetManagerUpdate() {
-    this.sortFacets();
+    // An update has to be forced for the facets to be visually updated, without being interacted with.
+    this.bindings.i18n.on('languageChanged', this.sortFacets);
```

Comment:
This component is different than the others, render doesn't do anything, so it's necessary to call sortFacets if we want an update

Reply:
They do, facet's by themselves outside of the manager re-render correctly.
The Stencil parent also have to be updated to see the visual change down to the children, and this component way of re-rendering is special (re-appends the children)